src/app/storage/components/AddEditStorageModal/ProviderForms/AzureForm.tsx

```diff
@@ -195,7 +205,69 @@ const InnerAzureForm = (props: IOtherProps & FormikProps<IFormValues>) => {
           </Button>
                 </Box>
             </Flex>
+            {isPopUpModalOpen &&
+                <Modal
+                    isSmall
+                    title="Repository information required"
+                    isOpen={isPopUpModalOpen}
+                    onClose={handleModalToggle}
+                    actions={[
+                        <Button key="confirm" variant="primary" onClick={handleModalToggle}>
+                            Close
+                        </Button>,
+                    ]}
+                    isFooterLeftAligned
+                >
+                    <Flex flexDirection="column">
+                        <Box my={1}>
+                            <p>
+                                To add an Azure repository, you will need the following information:
+                        </p>
+
+                        </Box>
+                        <Box my={1}>
+                            <ul>
+                                <li>
+                                    - Azure subscription ID
+                                </li>
+                                <li>
+                                    - Azure storage account
+                                </li>
+                                <li>
+                                    - Azure tennant ID
```

Comment:
typo. `Tenant`

Reply:
I don't think that should be capitalized